snuba/admin/clickhouse/querylog.py

```diff
@@ -39,5 +40,9 @@ def __run_querylog_query(query: str) -> ClickhouseResult:
     connection = get_ro_query_node_connection(
         StorageKey.QUERYLOG.value, ClickhouseClientSettings.QUERY
     )
-    query_result = connection.execute(query=query, with_column_types=True)
+    query_result = connection.execute(
+        query=query,
+        with_column_types=True,
+        settings={"max_threads": state.get_config("admin.querylog_threads", 4)},
```

Comment:
please cap this value. 
If anybody can set whatever they want, they can set a value that is large enough to use enough CPU to trip the alert that would page the ops on-call.

Reply:
good call, done